### PR TITLE
rm default namespace from demo notebooks

### PR DESCRIPTION
**Pull request by KPostOffice**
# Issue link
https://issues.redhat.com/browse/RHOAIENG-7554

# What changes have been made
Remove namespace param from example notebooks

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [x] Testing is not required for this change

### CODE REVIEW

**Comment by franciscojavierarceo** [comment]
Tests are failing

File: demo-notebooks/additional-demos/local_interactive.ipynb [hunk 1]
```diff
@@ -50,10 +50,9 @@
    "source": [
     "# Create and submit our cluster\n",
     "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\" unless you specify the local queue manually below\n",
-    "namespace = \"default\" # Update to your namespace\n",
     "cluster_name = \"hfgputest-1\"\n",
     "\n",
-    "cluster = Cluster(ClusterConfiguration(namespace=namespace,\n",
```
Comment (Bobbins228) [inline demo-notebooks/additional-demos/local_interactive.ipynb:56]: For this demo and basic interactive the namespace variable is required later for `generate_cert.generate_tls_cert(cluster_name, namespace)`
We could still remove the namespace variable in favour of `generate_cert.generate_tls_cert(cluster_name, cluster.namespace)` WDYT?

**Review by Bobbins228 — CHANGES_REQUESTED** [review]

**Review by Bobbins228 — COMMENTED** [review]
/lgtm

**Comment by Bobbins228** [comment]
Just a rebase and I am happy to approve

File: demo-notebooks/additional-demos/local_interactive.ipynb [hunk 2]
```diff
@@ -118,8 +117,8 @@
    "source": [
     "from codeflare_sdk import generate_cert\n",
     "\n",
-    "generate_cert.generate_tls_cert(cluster_name, namespace)\n",
-    "generate_cert.export_env(cluster_name, namespace)"
+    "generate_cert.generate_tls_cert(cluster_name, cluster.namespace)\n",
+    "generate_cert.export_env(cluster_name, cluster.namespace)"
```
Comment (Bobbins228) [inline demo-notebooks/additional-demos/local_interactive.ipynb:121]:
```suggestion
    "generate_cert.export_env(cluster_name, cluster.config.namespace)"
```
My bad. It should be `cluster.config.namespace` for these nbs

**Review by Bobbins228 — DISMISSED** [review]

**Review by Bobbins228 — COMMENTED** [review]
/lgtm

**Pull request merged**